WebHostLib/static/assets/player-options.js

```diff
@@ -194,13 +194,13 @@ const buildOptionsTable = (options, romOpts = false) => {
         element.appendChild(randomButton);
         break;
 
-      case 'special_range':
+      case 'named_range':
         element = document.createElement('div');
-        element.classList.add('special-range-container');
+        element.classList.add('named-range-container');
```

Comment:
I think you forgot to update the .css file that defines this and `named-range-wrapper`. The layout is broken for me in webhost. I can send a screenshot if it isn't broken for you for some reason.